test/FunctionalTests/Tests/DataOData/Tests/OData.Reader.Tests/JsonLight/FeedReaderJsonLightTests.cs

```diff
@@ -192,7 +192,7 @@ public void FeedReaderTest()
                 });
         }
 
-        [TestMethod, TestCategory("Reader.Json"), Variation(Description = "Verifies correct error behavior for invalid feeds in Json Light.")]
+//        [TestMethod, TestCategory("Reader.Json"), Variation(Description = "Verifies correct error behavior for invalid feeds in Json Light.")]

```

Comment:
>   [TestMethod, TestCategor [](start = 7, length = 27)

revert.

Reply:
Opened [https://github.com/OData/odata.net/issues/1427](https://github.com/OData/odata.net/issues/1427) to track re-enabling disabled tests.

---
In reply to: [259089699](https://github.com/OData/odata.net/pull/1204#discussion_r259089699) [](ancestors = 259089699)